### ISSUE TITLE
🐛 Bishop `R` is normalised but not re-orthogonalised, leaving ~1e-11 in `R Rᵀ − I`

### ISSUE THREAD

**Issue by nstarman** [GalacticDynamics/coordinax#952]
https://github.com/GalacticDynamics/coordinax/blob/main/packages/coordinaxs.curveframes/src/coordinaxs/curveframes/_src/bishop.py#L551

Both the scalar solve (`bishop.py:551`) and the batched one (`bishop.py:709`) finish with

```python
return U1_val / jnp.linalg.norm(U1_val)
```

Normalising restores `|U1| = 1` but not `U1 ⟂ T`: the solver's `U1·T` residual survives, `U2 = T × U1` inherits it, and so `R Rᵀ ≠ I` at solver tolerance. Measured on `main` (diffrax, `rtol = atol = 1e-10`): `max|R Rᵀ − I|` is **1.6e-12 … 1.8e-11** depending on τ, against ~5e-16 when `U1` is re-projected.

The residual is acknowledged in a comment at `bishop.py:66-68` rather than fixed:

> orthonormality of the resulting R holds to ~9e-12 over |tau| <= 60 on a helix, against ~9e-9 for the odeint solve this replaces

Tightening the tolerance made it 500× better, which is a real improvement, but it treats a structural property as a numerics problem. `Rᵀ` is then only an approximate inverse, so a `Alice → curve frame → Alice` position round trip carries the same error, where Frenet–Serret round-trips at 2.2e-16.

### Fix

The helper already exists — `_orthonormalize(v, T0_val)` at `bishop.py:153`, used for the user-supplied `normal_0` at `:500`. Call it in place of the bare division at both sites:

```python
return _orthonormalize(U1_val, tangent_fn(tau).value)
```

Measured effect of re-projecting: `max|R Rᵀ − I|` drops to 2.2e-16 at every τ tested (1, 5, 20, 100, 500). The tangent is recomputed a line later in `rotation_matrix_fn` anyway, so it can be hoisted rather than evaluated twice.

Worth keeping in the comment afterwards: the drift does **not** accumulate with arc length (τ = 500 is no worse than τ = 1), and the transport is genuinely rotation-minimising (`U1'·U2 ≈ 4e-8`, `U2'·U1 ≈ -4e-8`, equal and opposite) — both verified.
